### PR TITLE
doc: update example for `oras push`

### PR DESCRIPTION
**Pull request by qweeah**
This PR updates example for `oras push` command.

### CODE REVIEW

**Review by shizhMSFT — COMMENTED** [review]
The change LGTM but it seems we miss an example of setting artifact type.

File: cmd/oras/root/push.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ Example - Push multiple files with different media types:
   oras push localhost:5000/hello:v1 hi.txt:application/vnd.me.hi bye.txt:application/vnd.me.bye
 
 Example - Push file "hi.txt" with config type "application/vnd.me.config":
-  oras push --artifact-type application/vnd.me.config localhost:5000/hello:v1 hi.txt
+  oras push --image-spec v1.0 --artifact-type application/vnd.me.config localhost:5000/hello:v1 hi.txt
```
Comment (Wwwsylvia) [inline cmd/oras/root/push.go:72]: This seems to be the only example using `--artifact-type`. Do we need another example without `--image-spec v1.0`? Something like

```
Example - Push file "hi.txt" with artifact type "application/vnd.me.artifact":
  oras push --artifact-type application/vnd.me.artifact localhost:5000/hello:v1 hi.txt
```

Comment (qweeah) [reply cmd/oras/root/push.go:72]: added

File: cmd/oras/root/push.go [hunk 2]
```diff
@@ -69,7 +69,10 @@ Example - Push multiple files with different media types:
...
+
+Example - Push file "hi.txt" with artifact type "application/vnd.example+type":
+  oras push --artifact-type application/vnd.example+type localhost:5000/hello:v1 hi.txt
```
Comment (shizhMSFT) [inline cmd/oras/root/push.go:75]: This example of artifact type should be before the example of config type.

**Review by Wwwsylvia — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**